modules/jvm/bash/artifacts/opt/jboss/container/java/jvm/java-default-options

```diff
@@ -135,7 +135,13 @@ gc_config() {
   local maxHeapFreeRatio=${GC_MAX_HEAP_FREE_RATIO:-20}
   local timeRatio=${GC_TIME_RATIO:-4}
   local adaptiveSizePolicyWeight=${GC_ADAPTIVE_SIZE_POLICY_WEIGHT:-90}
-  local gcOptions="${GC_CONTAINER_OPTIONS:--XX:+UseParallelOldGC}"
+
+  if [ "$JAVA_VERSION" -lt 17 ]; then
```

Comment:
I'm actually not sure yet whether we should solve this problem in this way, or introduce a parallel variant of this cekit module.

Reply:
Why those gymnastics? AFAIK, just using `-XX:+UseParallelGC` will work for JDK 17, 11 and 8 and looking at the enabled flags via `-XX:+Use[...] -XX:+PrintFlagsFinal` for JDK 8 and 11 those are equivalent.

Reply:
Thank you, you're right. I was going from the docs as to the meaning of the flags, rather than the impact it had on the final flags. Re-reading "JEP 366: Deprecate the ParallelScavenge + SerialOld GC Combination" I still feel like the UseParallelOldGC flag is written backwards... but comparing the flag differences between all three JDK versions with the options enabled/disabled is good enough for me. Thanks for the tip on comparing flags like that.